src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -929,17 +955,17 @@ public SearchPublisher getSearch() {
 
   @Override
   public void setLocale(String code) {

```

Comment:
Setting locale on the ProgramAB service changes it for the entire platform? this surly can't be correct.

Reply:
it's preserving the same interface as before, so consider it a helper - or consider the back end has finally been normalized